kroxylicious-krpc-plugin/src/main/java/io/kroxylicious/krpccodegen/schema/MessageSpec.java

```diff
@@ -124,16 +138,13 @@ public List<RequestListenerType> listeners() {
     }
 
     public String dataClassName() {
-        switch (type) {
-            case HEADER:
-            case REQUEST:
-            case RESPONSE:
-                // We append the Data suffix to request/response/header classes to avoid
-                // collisions with existing objects. This can go away once the protocols
-                // have all been converted and we begin using the generated types directly.
-                return struct.name() + "Data";
-            default:
-                return struct.name();
-        }
+        return switch (type) {
+            case HEADER, REQUEST, RESPONSE ->
+                    // We append the Data suffix to request/response/header classes to avoid
+                    // collisions with existing objects. This can go away once the protocols
+                    // have all been converted and we begin using the generated types directly.
```

Comment:
Are we ever going to be in a position to drop the `Data` suffix?

Reply:
The class is copied from Apache Kafka.  I refreshed updated the class as part of the upgrade.  The comment is theirs.